tests/integration/fileTypeTests.py

```diff
@@ -97,19 +66,18 @@ def setUpClass(cls):
         cls.jobIdDict = jobIdDict
 
     @staticmethod
-    def load_definitions(interfaces, force_tas_load):
+    def load_definitions(interfaces, force_tas_load, ruleList = None):
         """Load file definitions."""
         SchemaLoader.loadAllFromPath(join(CONFIG_BROKER["path"],"dataactvalidator","config"))
         SQLLoader.loadSql("sqlRules.csv")
 
-        # @todo: here is a stop-gap solution; we've added new rules which are
-        # filtering out previously usable test data. Temporarily remove these
-        # rules
-        to_delete = interfaces.validationDb.session.query(RuleSql).filter(
-            RuleSql.rule_label.in_(('A30', 'A32', 'A33', 'B20', 'B19', 'C19')))
-        for rule in to_delete:
-            interfaces.validationDb.session.delete(rule)
-        interfaces.validationDb.session.commit()
+        if ruleList is not None:
+            # If rule list provided, drop all other rules
+            to_delete = interfaces.validationDb.session.query(RuleSql).filter(not_(
+                RuleSql.rule_label.in_(ruleList)))
```

Comment:
Nice--thanks for putting the list somewhere more universal! I like the way this is implemented as strictly a test thing w/o adding another param to the application's sqlloader.